packages/account/src/account.ts

```diff
@@ -244,21 +244,21 @@ export class Account extends AbstractAccount {
   }
 
   /**
-   * Adds resources to the transaction enough to fund it.
+   * Funds a transaction request by adding the necessary resources.
    *
-   * @param request - The transaction request.
-   * @param requiredQuantities - The coin quantities required to execute the transaction.
-   * @param fee - The estimated transaction fee.
-   * @returns A promise that resolves when the resources are added to the transaction.
+   * @template T - The type of the TransactionRequest.
+   * @param T - request - The transaction request to fund.
```

Comment:
Do these two render correctly with Typedoc?

Reply:
@arboleya I believe so because it is [documented](https://typedoc.org/tags/template/)